getpaid/registry.py

```diff
@@ -0,0 +1,56 @@
+import importlib
+
+from django.conf.urls import include, url
+
+from getpaid.processor import BaseProcessor
+
+
+def importable(name):
+    try:
+        importlib.import_module(name)
+        return True
+    except ImportError:
+        return False
+
+
+class PluginRegistry(object):
```

Comment:
Maybe we could do one of:
* Use an existing plugin plugin like https://pythonhosted.org/django-plugins/
* Use entry_points
* Create a separate package of this